tokio-console/src/view/mod.rs

```diff
@@ -167,6 +167,31 @@ impl View {
                         self.state = ResourcesList;
                         update_kind = UpdateKind::Other;
                     }
+                    key!(Enter) => {
+                        if let Some(op) = view.async_ops_table.selected_item().upgrade() {
+                            if let Some(task_id) = op.borrow().task_id() {
+                                let task: u64 = task_id.to_string().parse().unwrap();
+                                let item = self
+                                    .tasks_list
+                                    .sorted_items
+                                    .iter()
+                                    .filter_map(|i| i.upgrade())
+                                    .find(|t| {
+                                        let task_id: u64 =
+                                            t.borrow().id().to_string().parse().unwrap();
+                                        task_id == task
+                                    });
```

Comment:
The `Id<Task>` can be compared directly, so it doesn't need to be turned into a string and then parsed to a u64.

Also, let's call the result `task`, as I think it's clearer than `item`.

```suggestion
                                let task = self
                                    .tasks_list
                                    .sorted_items
                                    .iter()
                                    .filter_map(|i| i.upgrade())
                                    .find(|t| task_id == t.borrow().id());
```